activate

```
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Add `-e`. Scripts should fail fast.